pkg/state/scripts_storage.go

```diff
@@ -449,3 +449,11 @@ func (ss *scriptsStorage) reset() {
 	ss.assetScriptsHasher.reset()
 	ss.accountScriptsHasher.reset()
 }
+
+func (ss *scriptsStorage) AccountScriptsHasher() *stateHasher {
```

Comment:
Should not be public, used only in state

Reply:
Fixed